recipe-box-frontend/src/main/frontend/src/components/Header/Header.tsx

```diff
@@ -23,41 +24,46 @@ const Header: React.FC = () => {
                                 <li >
                                 {userType === UserType.CHEF && (
                                     <>
-                                        <button 
-                                        className="nav-item" 
-                                        onClick={() => setIsOpen(!isOpen)}
-                                        >
-                                        Recipes
-                                        <ChevronDown className='chevron-item'></ChevronDown>
-                                        </button>
-                                        
-                                        
-                                        {isOpen && (
-                                        <ul className="dropdown-menu" onClick={() => setIsOpen(!isOpen)}>
-                                            <li><Link to="/recipes/list" className="dropdown-item">Public Recipes</Link></li>
-                                            <li><Link to="/recipes/myRecipes" className="dropdown-item">My Recipes</Link></li>
-                                            <li><Link to="/recipes/ingredients" className="dropdown-item">Recipes with My Ingredients</Link></li>
-                                        </ul>
-                                        )}
-                                    </>
+                                    <ul className='left'>
+                                    <Dropdown buttonText="Recipes" content={
+                                        <>
+                                            <button className="dropdown-item" onClick={() => navigate("/recipes/list")}>Public recipes</button>
```

Comment:
Why use navigate when you could use the Link component?

Reply:
Because Alin said in a review: "Also, we are using "useNavigate" throughout our app. For consistency, you should use it here too.", so i changed the links to be "useNavigate" instead. Should i change them back?

Reply:
:)) There aren't such things as oopsies in programming. While you are right that it would be recommended to keep in consistency, the Link element would be a more preferable method than useNavigate for navigating to the pages. You can eventually use useNavigate on the pages that should be accessed only by a specific group of users in order to redirect those who shouldn't have access to the home page.
@en-marius-gociman , @en-catrinel-lupu, correct me if I'm wrong